internal/storage/service_test.go

```diff
@@ -569,6 +569,67 @@ func TestServiceList(t *testing.T) {
 	})
 }
 
+func TestServiceListAips(t *testing.T) {
+	t.Parallel()
+
+	t.Run("Returns defined AIPs", func(t *testing.T) {
+		t.Parallel()
+
+		ctx := context.Background()
+		attrs := &setUpAttrs{}
+		svc := setUpService(t, attrs)
+
+		payload := &goastorage.ListAipsPayload{
+			Limit: ref.New(20),
+		}
+		aips := &goastorage.AIPs{
+			Items: goastorage.AIPCollection{
+				{
+					Name:      "Test AIP 1",
+					UUID:      aipID,
+					ObjectKey: objectKey,
+					Status:    "stored",
+					CreatedAt: time.Now().Format(time.RFC3339),
+				},
+			},
+			Page: &goastorage.EnduroPage{
+				Limit:  20,
+				Offset: 0,
+				Total:  1,
+			},
+		}
+
+		attrs.persistenceMock.
+			EXPECT().
+			ListAIPs(ctx, payload).
+			Return(aips, nil).
+			Times(1)
```

Comment:
`Times(1)` is the default, it doesn't need to be set explicitly.

Reply:
Copying without looking, there was A LOT of `.Times(1)` in these test :grin: